### DISCUSSION TITLE
SEO

### DISCUSSION THREAD

**Discussion by ellielo** [NAIST-SE/AttractingContributors#306] [General]
What are the best seo tools in your opinion? How do you become professional in using them?

**Comment by XelaAl** [comment]
There is a great number of SEO tools. For me, the best ones are Google Trends, Ahrefs Keyword Generator, Ubersuggest, Google Search Console, Bing Webmaster Tools, Google Analytics.

**Comment by hernessteve** [comment]
SEranking, MozBar, SEMrush, Ahrefs.
There are a lot of seo tools, it all depends what purposes do you need them for. You can start learning about them online of course, youtube and google has a lot of useful information of course. There is also a great [semrush training video](https://www.udemy.com/join/login-popup/?next=/course/ultimate-semrush-course/learn/lecture/28925120#overview) you can watch for starters. If you think it is something you are really interested in the next step would be to take some paid courses.
